<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>calc</class>
 <widget class="QMainWindow" name="calc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>593</width>
    <height>411</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calculator</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(48, 56, 68);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="5">
     <widget class="QPushButton" name="buttonPOW">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #C4C4C4;
	border-style: solid;
	border: 2px;
	padding: 20px;	
	border-radius:8px;
	color: black;
}

QPushButton:pressed{
	background-color: #A9A9A9;
	border-style: solid;
	border: 2px;
	padding: 20px;	
	border-radius:8px;
	color: black;
}</string>
      </property>
      <property name="text">
       <string>X^Y</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="button9">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #C4C4C4;
	border-style: solid;
	border: 2px;
	padding: 20px;	
	border-radius:8px;
	color: black;
}

QPushButton:pressed{
	background-color: #A9A9A9;
	border-style: solid;
	border: 2px;
	padding: 20px;	
	border-radius:8px;
	color: black;
}</string>
      </property>
      <property name="text">
       <string>9</string>
      </property>
     </widget>
    </item>
    <item row="5" column="5">
     <widget class="QPushButton" name="buttonLOG">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #C4C4C4;
	border-style: solid;
	border: 2px;
	padding: 20px;	
	border-radius:8px;
	color: black;
}

QPushButton:pressed{
	background-color: #A9A9A9;
	border-style: solid;
	border: 2px;
	padding: 20px;	
	border-radius:8px;
	color: black;
}</string>
      </property>
      <property name="text">
       <string>log</string>
      </property>
     </widget>
    </item>
    <item row="5" column="3" colspan="2">
     <widget class="QPushButton" name="buttonEQUAL">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #2E36F8;
	border: 2px black;
	padding: 20px;	
	color: black;
}

QPushButton:pressed{
	background-color: #5257fa;
	border: 2px black;
	padding: 20px;	
	color: black;
}</string>
      </property>
      <property name="text">
       <string>=</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="button1">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #C4C4C4;
	border-style: solid;
	border: 2px;
	padding: 20px;	
	border-radius:8px;
	color: black;
}

QPushButton:pressed{
	background-color: #A9A9A9;
	border-style: solid;
	border: 2px;
	padding: 20px;	
	border-radius:8px;
	color: black;
}</string>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="button8">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #C4C4C4;
	border-style: solid;
	border: 2px;
	padding: 20px;	
	border-radius:8px;
	color: black;
}

QPushButton:pressed{
	background-color: #A9A9A9;
	border-style: solid;
	border: 2px;
	padding: 20px;	
	border-radius:8px;
	color: black;
}</string>
      </property>
      <property name="text">
       <string>8</string>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QPushButton" name="buttonADD">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #C4C4C4;
	border-style: solid;
	border: 2px;
	padding: 20px;	
	border-radius:8px;
	color: black;
}

QPushButton:pressed{
	background-color: #A9A9A9;
	border-style: solid;
	border: 2px;
	padding: 20px;	
	border-radius:8px;
	color: black;
}</string>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="button2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #C4C4C4;
	border-style: solid;
	border: 2px;
	padding: 20px;	
	border-radius:8px;
	color: black;
}

QPushButton:pressed{
	background-color: #A9A9A9;
	border-style: solid;
	border: 2px;
	padding: 20px;	
	border-radius:8px;
	color: black;
}</string>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QPushButton" name="button0">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #C4C4C4;
	border-style: solid;
	border: 2px;
	padding: 20px;	
	border-radius:8px;
	color: black;
}

QPushButton:pressed{
	background-color: #A9A9A9;
	border-style: solid;
	border: 2px;
	padding: 20px;	
	border-radius:8px;
	color: black;
}</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QPushButton" name="buttonAC">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #C4C4C4;
	border-style: solid;
	border: 2px;
	padding: 20px;	
	border-radius:8px;
	color: black;
}

QPushButton:pressed{
	background-color: #A9A9A9;
	border-style: solid;
	border: 2px;
	padding: 20px;	
	border-radius:8px;
	color: black;
}</string>
      </property>
      <property name="text">
       <string>AC</string>
      </property>
     </widget>
    </item>
    <item row="3" column="5">
     <widget class="QPushButton" name="buttonSQRT">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #C4C4C4;
	border-style: solid;
	border: 2px;
	padding: 20px;	
	border-radius:8px;
	color: black;
}

QPushButton:pressed{
	background-color: #A9A9A9;
	border-style: solid;
	border: 2px;
	padding: 20px;	
	border-radius:8px;
	color: black;
}</string>
      </property>
      <property name="text">
       <string>âˆš</string>
      </property>
     </widget>
    </item>
    <item row="3" column="4">
     <widget class="QPushButton" name="buttonDIV">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #C4C4C4;
	border-style: solid;
	border: 2px;
	padding: 20px;	
	border-radius:8px;
	color: black;
}

QPushButton:pressed{
	background-color: #A9A9A9;
	border-style: solid;
	border: 2px;
	padding: 20px;	
	border-radius:8px;
	color: black;
}</string>
      </property>
      <property name="text">
       <string>/</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="button7">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #C4C4C4;
	border-style: solid;
	border: 2px;
	padding: 20px;	
	border-radius:8px;
	color: black;
}

QPushButton:pressed{
	background-color: #A9A9A9;
	border-style: solid;
	border: 2px;
	padding: 20px;	
	border-radius:8px;
	color: black;
}</string>
      </property>
      <property name="text">
       <string>7</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="button4">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #C4C4C4;
	border-style: solid;
	border: 2px;
	padding: 20px;	
	border-radius:8px;
	color: black;
}

QPushButton:pressed{
	background-color: #A9A9A9;
	border-style: solid;
	border: 2px;
	padding: 20px;	
	border-radius:8px;
	color: black;
}</string>
      </property>
      <property name="text">
       <string>4</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QPushButton" name="buttonMUL">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #C4C4C4;
	border-style: solid;
	border: 2px;
	padding: 20px;	
	border-radius:8px;
	color: black;
}

QPushButton:pressed{
	background-color: #A9A9A9;
	border-style: solid;
	border: 2px;
	padding: 20px;	
	border-radius:8px;
	color: black;
}</string>
      </property>
      <property name="text">
       <string>X</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="button5">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #C4C4C4;
	border-style: solid;
	border: 2px;
	padding: 20px;	
	border-radius:8px;
	color: black;
}

QPushButton:pressed{
	background-color: #A9A9A9;
	border-style: solid;
	border: 2px;
	padding: 20px;	
	border-radius:8px;
	color: black;
}</string>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QPushButton" name="buttonDEL">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #C4C4C4;
	border-style: solid;
	border: 2px;
	padding: 20px;	
	border-radius:8px;
	color: black;
}

QPushButton:pressed{
	background-color: #A9A9A9;
	border-style: solid;
	border: 2px;
	padding: 20px;	
	border-radius:8px;
	color: black;
}</string>
      </property>
      <property name="text">
       <string>DEL</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="button6">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #C4C4C4;
	border-style: solid;
	border: 2px;
	padding: 20px;	
	border-radius:8px;
	color: black;
}

QPushButton:pressed{
	background-color: #A9A9A9;
	border-style: solid;
	border: 2px;
	padding: 20px;	
	border-radius:8px;
	color: black;
}</string>
      </property>
      <property name="text">
       <string>6</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="6">
     <widget class="QLineEdit" name="display">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
	background-color: gray;
	border: 2px solid gray;
	color: 000000;
}</string>
      </property>
     </widget>
    </item>
    <item row="4" column="5">
     <widget class="QPushButton" name="buttonFACT">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #C4C4C4;
	border-style: solid;
	border: 2px;
	padding: 20px;	
	border-radius:8px;
	color: black;
}

QPushButton:pressed{
	background-color: #A9A9A9;
	border-style: solid;
	border: 2px;
	padding: 20px;	
	border-radius:8px;
	color: black;
}</string>
      </property>
      <property name="text">
       <string>!</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QPushButton" name="button3">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #C4C4C4;
	border-style: solid;
	border: 2px;
	padding: 20px;	
	border-radius:8px;
	color: black;
}

QPushButton:pressed{
	background-color: #A9A9A9;
	border-style: solid;
	border: 2px;
	padding: 20px;	
	border-radius:8px;
	color: black;
}</string>
      </property>
      <property name="text">
       <string>3</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QPushButton" name="buttonDOT">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #C4C4C4;
	border-style: solid;
	border: 2px;
	padding: 20px;	
	border-radius:8px;
	color: black;
}

QPushButton:pressed{
	background-color: #A9A9A9;
	border-style: solid;
	border: 2px;
	padding: 20px;	
	border-radius:8px;
	color: black;
}</string>
      </property>
      <property name="text">
       <string>.</string>
      </property>
     </widget>
    </item>
    <item row="4" column="4">
     <widget class="QPushButton" name="buttonSUB">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #C4C4C4;
	border-style: solid;
	border: 2px;
	padding: 20px;	
	border-radius:8px;
	color: black;
}

QPushButton:pressed{
	background-color: #A9A9A9;
	border-style: solid;
	border: 2px;
	padding: 20px;	
	border-radius:8px;
	color: black;
}</string>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="buttonHELP">
      <property name="text">
       <string>Help...</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
